integreat_cms/cms/templates/tutorials/page_access_statistics.html

```diff
@@ -0,0 +1,38 @@
+{% extends "../generic_tutorial_dialog.html" %}
+{% load i18n %}
+{% load static %}
+{% block content %}
+    <!-- Pop-Up Header -->
+    <div class="flex items-center bg-water-500 w-full rounded h-12">
+        <h2 class="flex items-center font-semibold leading-normal pl-2 text-black text-xl">
```

Comment:
```suggestion
        <h2 class="flex items-center font-semibold leading-normal text-black text-lg">
```
I think `xl` is a bit too large. I think `lg` is a bit closer to the intended 22px in the design :)